api/src/main/java/org/infinispan/api/InfinispanManager.java

```diff
@@ -0,0 +1,29 @@
+package org.infinispan.api;
+
+import org.infinispan.api.configuration.InfinispanClientConfig;
+import org.infinispan.api.configuration.InfinispanConfig;
+
+/**
+ * Entry point for Infinispan API - client
+ *
+ * @author Katia Aresti, karesti@redhat.com
+ * @since 10.0
+ */
+@Experimental
+public class InfinispanManager {
+
+   public static final Infinispan newInfinispanClient(InfinispanClientConfig config) {
```

Comment:
Regardless of the location of these two methods, I think it would be good to reduce their verbosity. So maybe:

```java
 public static final Infinispan client(InfinispanClientConfig config) {..}
 public static final Infinispan embedded(InfinispanConfig config) {..}
```

Or if we want it to be more explicit, just remove the superfluous `infinispan` as we're already calling this from `InfinispanManager`, e.g. `newClient`

Reply:
We could maybe go even further and rename the configs to just `ClientConfig` and `EmbeddedConfig`?

Reply:
yes